pydevices/HtsDevices/acq2106_435sc.py

```diff
@@ -81,80 +82,83 @@ def getUUT(self):
         uut = acq400_hapi.Acq2106(self.node.data(), monitor=False, has_wr=True)
         return uut
 
+    def computeGains(self, g):
+        g1opts = [ 1000, 100, 10, 1 ]
+        g2opts = [ 1, 2, 5, 10 ]
 
-    def setGainsOffsets(self, card):
+        for g1opt in g1opts:
+            if g >= g1opt:
+                g1 = g1opt
+                g2 = g // g1opt
+                if g2 not in g2opts:
+                    raise MDSplus.DevBAD_PARAMETER(
+                            "SC_GAIN must be computable from (one of: 1000, 100, 10, 1) * (one of: 1, 2, 5, 10) ; not %d" % (g,))
+                return (g1, g2)
+
+    def setGainsOffsets(self, site):
         import epics
         import socket
 
+        if site not in [1, 3, 5]:
+            # (slw) TODO: Improve
+            print('site is not 1, 3, or 5')
+            return
+
         domainName = socket.gethostbyaddr(str(self.node.data()))[0]
```

Comment:
We should add a note about needing to replace this with an EPICS_NAME node or somesuch

Reply:
This has been resolved in the Slowmon PR: a new tree node was created to input the EPICS name.